workitem/link/link_repository.go

```diff
@@ -97,12 +97,14 @@ func (r *GormWorkItemLinkRepository) ValidateCorrectSourceAndTargetType(ctx cont
 	return nil
 }
 
-// CheckParentExist returns error if there is an attempt to create more than 1 parent of a workitem.
-func (r *GormWorkItemLinkRepository) CheckParentExist(ctx context.Context, sourceID, targetID uint64, linkTypeID uuid.UUID) error {
+// CheckParentExists returns error if there is an attempt to create more than 1 parent of a workitem.
+func (r *GormWorkItemLinkRepository) CheckParentExists(ctx context.Context, sourceID, targetID uint64, linkTypeID uuid.UUID) (*bool, error) {
```

Comment:
@DhritiShikhar sorry. I did miss the `*bool` in @baijum 's proposal but I think it is overengineered for this task. If we would return a super big object, I'd agree to return a pointer. But not for a boolean.